source/standards/source-code/use-github.html.md.erb

```diff
@@ -6,17 +6,21 @@ review_in: 6 months
 
 # <%= current_page.data.title %>
 
-[GDS uses GitHub as its code repository](https://technology.blog.gov.uk/2016/05/31/how-we-use-git-at-the-government-digital-service/).
+GDS uses the [alphagov] organisation on [GitHub] to collaborate on code. The GOV.UK One Login programme uses [govuk-one-login].
 
-Put new repositories for GDS services in the [alphagov](https://github.com/alphagov/) organisation on GitHub.
+### Getting access to `alphagov`
```

Comment:
This leads me to questions like "what if I need access to `govuk-one-login`? Are there differences in terms of this guidance between `alphagov` and `govuk-one-login` and, if there are, why, and what are those difference? IMO we should have general guidance for the use of GitHub across GDS (regardless of org), and then separate sections for when something might differ (e.g. the process for requesting access).

Reply:
I actually don't know what the policy is about access to govuk-one-login org .. is it documented anywhere Rafal?

Reply:
Got something [documented in our Team Manual](https://github.com/govuk-one-login/team-manual/blob/main/source/new-starter-guide/github-access-management.html.md.erb) (happy to share the hosted page URL over Slack).

The things to point out that are different:
* We require SC for normal access
* If SC is not held, risk is recorded and accepted, resulting in individuals being assigned as outside collaborators
* We use IaC Terraform to manage the state of

Reply:
Rafal if you think we should link to guidance for govuk-one-login, please would you propose some suitable wording? 

I would like to get this change merged, and since the previous guidance didn't say anything about govuk-one-login, I have a slight preference for picking that up in a separate PR if that would be acceptable to others here?